.github/CODEOWNERS

```diff
@@ -1,5 +1,26 @@
 # Code owners file.
 # This file controls who is tagged for review for any given pull request.
+#
+# What is a "CODEOWNER"?
+#
+# A CODEOWNER MUST ____ for packages they own:
+# - introduce themselves at a Python SIG meeting
+# - demonstrate advanced knowledge of the dependencies
+# - respond to issues when they are notified
+# - fix failing unit tests or any other blockers to the CI/CD workflow
+# - update usage of `opentelemetry-python-core` APIs upon the introduction of breaking changes
```

Comment:
```suggestion
```
Hmm, not necesarily. The components of OpenTelemetry core will be just more dependencies for a contrib package, just like the instrumented libraries themselves. If a contrib package can limit the versions of the instrumented library that is supported (let's say, a certain contrib package only works for `2.x.y` versions of `Flask`) the contrib package can do the same for the OpenTelemetry core components.

Reply:
That makes sense. I guess the intention was that they be involved when those changes happen? Maybe I can say:

```suggestion
# - review PRs that update `opentelemetry-python-core` dependencies
```

but maybe that should just be covered by a general "review all PRs related to your package"?

Reply:
Yes, reviewing the PRs should be enough.